src/public/sass/ITCSS/06_components/pages/home/measurement-solutions/_swiper.scss

```
@@ -4,6 +4,11 @@
   width: 100%;
   height: 36vh;
 }
+
+.swiper-wrapper--center {
+  justify-content: center;
+}
+
```

```suggestion
.swiper-wrapper--center {
  @include phoneXL() {
    justify-content: center;
  }
}
```
This could solve the issue @CheloChidoone say.